### PR TITLE
Small improvements to the vagrant build for centos

### PR DESCRIPTION
**Pull request by akon-dey**

### CODE REVIEW

File: vagrant/centos/vagrant-build-gpdb.sh [hunk 1]
```diff
@@ -25,7 +25,7 @@ rm -f ~/.ssh/id_rsa
 rm -f ~/.ssh/id_rsa.pub
 rm -f ~/.ssh/authorized_keys
 ssh-keygen -t rsa -N "" -f "$HOME/.ssh/id_rsa"
-cat ~/.ssh/id_rsa.pub > ~/.ssh/authorized_keys
+cat ~/.ssh/id_rsa.pub >> ~/.ssh/authorized_keys
```
Comment (danielgustafsson) [inline vagrant/centos/vagrant-build-gpdb.sh]: Considering the file is removed on row 26, what is the problem you're trying to solve with this?

Comment (akon-dey) [inline vagrant/centos/vagrant-build-gpdb.sh]: Good point. We should also remove line 26 to prevent any previous authorized keys from being clobbered.

**Comment by ascherbaum-pivotal** [comment]
Please let us know once you signed the CLA.

Thanks,
Andreas

**Comment by hlinnaka** [comment]
@xinzweb, can you have a look at this, please? I don't know anything about the vagrant stuff.

**Comment by ascherbaum-pivotal** [comment]
Looks like sane improvements, especially the handling of the ssh keys.

**Comment by xinzweb** [comment]
+1

**Pull request closed without merging**